helix-view/src/editor.rs

```diff
@@ -249,9 +249,17 @@ pub struct Editor {
     pub idle_timer: Pin<Box<Sleep>>,
     pub last_motion: Option<Motion>,
 
+    pub last_completion: Option<CompleteAction>,
+
     pub exit_code: i32,
 }
 
+#[derive(Debug, Clone)]
```

Comment:
Does this need to be `Clone`?

Reply:
Yes, because of this `clone` https://github.com/helix-editor/helix/blob/master/helix-term/src/ui/editor.rs#L769